src/interpolations.jl

```diff
@@ -232,7 +232,7 @@ end
 Evaluate all shape function hessians, gradients and values of `ip` at once at the reference point `ξ`
 and store them in `hessians`, `gradients`, and `values`.
 """
-@propagate_inbounds function reference_shape_hessians_gradients_and_values!(hessians::AbstractVector, gradients::AbstractVector, values::AbstractVector, ip::Interpolation, ξ::Vec)
+@propagate_inbounds function reference_shape_hessians_gradients_and_values!(hessians::HAT, gradients::GAT, values::SAT, ip::IP, ξ::Vec) where {IP <: Interpolation, SAT <: AbstractArray, GAT <: AbstractArray, HAT <: AbstractArray}
```

Comment:
Why the new parameters (the `where` stuff)? They are not used in the function body?

Reply:
I was just asking myself why we do that for the `reference_shape_gradients_and_values!`, I think that case is a typo/unnecessary usage of `where` and we should rather change that (and also to `AbstractVector`).

Reply:
They were originally written like that in https://github.com/Ferrite-FEM/Ferrite.jl/pull/819 but I see no reason for it.